tests/passing/ifand.ml

```diff
@@ -1,13 +1,13 @@
 let _ =
   if cond1
-  && cond2
+     && cond2
```

Comment:
Yes, I disabled this special-case (and likewise for "when")... :/
Not completely sure about it, but it disagrees with JaneStreet's policies, and feels too specific to add a config option. I'd suggest putting the operator at the end of line instead, for alignment.